AppleStoreProject/AppleStoreProject/ForYouViewController.swift

```diff
@@ -7,4 +7,275 @@
+        recomendationLable.text = "Рекомендуется вам"
+        recomendationLable.font = .boldSystemFont(ofSize: 20)
+        view.addSubview(recomendationLable)
+    }
+
+    private func createInformationDeliveryImage() {
+        informationDeliveryImage.frame = CGRect(x: 50, y: 550, width: 40, height: 40)
+        informationDeliveryImage.image = UIImage(systemName: "app.badge")
+        informationDeliveryImage.tintColor = .systemRed
+        view.addSubview(informationDeliveryImage)
+    }
+
+    private func createInformationDeliveryLable() {
+        informationDeliveryLable.frame = CGRect(x: 120, y: 515, width: 200, height: 100)
+        informationDeliveryLable.text = "Получайте новости о своем заказе в режиме реального времени."
+        informationDeliveryLable.numberOfLines = 3
+        informationDeliveryLable.lineBreakMode = .byWordWrapping
+        informationDeliveryLable.font = .boldSystemFont(ofSize: 14)
+        view.addSubview(informationDeliveryLable)
+    }
+
+    private func createNotificationDeliveryLable() {
+        notificationDeliveryLable.frame = CGRect(x: 120, y: 575, width: 260, height: 100)
+        notificationDeliveryLable.text = "Включите уведомления, чтобы получать новости о своем заказе."
+        notificationDeliveryLable.numberOfLines = 2
+        notificationDeliveryLable.lineBreakMode = .byWordWrapping
+        notificationDeliveryLable.font = .boldSystemFont(ofSize: 13)
+        notificationDeliveryLable.textColor = .systemGray
+        view.addSubview(notificationDeliveryLable)
+    }
+
+    private func createNotificationDeliveryImage() {
+        notificationDeliveryImage.frame = CGRect(x: 365, y: 577, width: 15, height: 15)
+        notificationDeliveryImage.image = UIImage(systemName: "chevron.right")
+        notificationDeliveryImage.tintColor = .systemGray
+        view.addSubview(notificationDeliveryImage)
+    }
+
+    private func createnotificationDeliveryUnderLineView() {
+        notificationDeliveryUnderLineView.frame = CGRect(x: 15, y: 660, width: 365, height: 1)
+        notificationDeliveryUnderLineView.backgroundColor = #colorLiteral(red: 0.9373239875, green: 0.9317518473, blue: 0.941607058, alpha: 1)
+        view.addSubview(notificationDeliveryUnderLineView)
+    }
+
+    private func createyourDevicesLable() {
+        yourDevicesLable.frame = CGRect(x: 30, y: 700, width: 270, height: 40)
+        yourDevicesLable.text = "Ваши устройства"
+        yourDevicesLable.font = .boldSystemFont(ofSize: 25)
+        view.addSubview(yourDevicesLable)
+    }
+
+    private func  createShowDevicesButton() {
+        showDevicesButton.frame = CGRect(x: 280, y: 710, width: 120, height: 20)
+        showDevicesButton.setTitle("Показать все", for: .normal)
+        showDevicesButton.setTitleColor(UIColor.systemBlue, for: .normal)
+        view.addSubview(showDevicesButton)
+    }
+
+    private let imagePicker = UIImagePickerController()
```

Comment:
```suggestion
    private let imagePicker = UIImagePickerController()
```
что мы тут забыли его на верх